plugins/Geant4Output2EDM4hep_DRC.cpp

```diff
@@ -0,0 +1,689 @@
+#ifndef DD4HEP_DDG4_GEANT4OUTPUT2EDM4hep_DRC_H
+#define DD4HEP_DDG4_GEANT4OUTPUT2EDM4hep_DRC_H
+
+///  Framework include files
+#include <DD4hep/Detector.h>
+#include <DDG4/EventParameters.h>
+#include <DDG4/Geant4OutputAction.h>
+#include <DDG4/RunParameters.h>
+
+#include "FiberDRCaloSDAction.h"
+
+/// edm4hep include files
+#include <edm4hep/MCParticleCollection.h>
+#include <edm4hep/SimTrackerHitCollection.h>
+#include <edm4hep/CaloHitContributionCollection.h>
+#include <edm4hep/SimCalorimeterHitCollection.h>
+#include "edm4hep/RawCalorimeterHitCollection.h"
+#include "edm4hep/RawTimeSeriesCollection.h"
+#include <edm4hep/EDM4hepVersion.h>
+/// podio include files
+#include <podio/Frame.h>
+#if PODIO_VERSION_MAJOR > 0 || (PODIO_VERSION_MAJOR == 0 && PODIO_VERSION_MINOR >= 99)
+#include <podio/ROOTWriter.h>
+#else
+#include <podio/ROOTFrameWriter.h>
+#endif
+#include <podio/podioVersion.h>
+
+/// Namespace for the AIDA detector description toolkit
+namespace dd4hep {
+
+  class ComponentCast;
+
+  /// Namespace for the Geant4 based simulation part of the AIDA detector description toolkit
+  namespace sim {
+
+    class  Geant4ParticleMap;
+    class Geant4Output2EDM4hep_DRC : public Geant4OutputAction  {
+    protected:
+#if PODIO_VERSION_MAJOR > 0 || (PODIO_VERSION_MAJOR == 0 && PODIO_VERSION_MINOR >= 99)
```

Comment:
```suggestion
#if PODIO_BUILD_VERSON >= PODIO_VERSION(0, 99, 0)
```
Easier to read and will definitely do what you want